packages/xr-webxr/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@galacean/engine-xr-webxr",
+  "version": "1.1.0-alpha.0",
+  "publishConfig": {
+    "access": "public",
+    "registry": "https://registry.npmjs.org"
+  },
+  "license": "MIT",
+  "main": "dist/main.js",
+  "module": "dist/module.js",
+  "debug": "src/index.ts",
+  "browser": "dist/browser.js",
+  "types": "types/index.d.ts",
+  "scripts": {
+    "b:types": "tsc"
+  },
+  "umd": {
+    "name": "Galacean.webxr",
+    "globals": {
+      "@galacean/engine": "Galacean"
+    }
+  },
+  "files": [
+    "dist/**/*",
+    "libs/**/*",
+    "types/**/*"
+  ],
+  "devDependencies": {
+    "@galacean/engine-design": "workspace:*",
+    "@galacean/engine-xr": "workspace:*",
+    "@galacean/engine": "workspace:*"
+  },
```

Comment:
If XR Interface is in  `engine-xr`, you only dependencies `@galacean/engine-xr` and `@galacean/engine-math`